requirements.txt

```diff
@@ -6,7 +6,7 @@ intervaltree
 progressbar2
 pyyaml
 -e third_party/prjxray
-git+https://github.com/SymbiFlow/fasm.git#egg=fasm
+git+https://github.com/SymbiFlow/fasm.git@052f629#egg=fasm
```

Comment:
Let's drop this `@`, and fix the upstream issue.

Reply:
@HackerFoo has fixed this I believe?

Reply:
After bumping `prjxray` submodule the version locking was no longer necessary.